students/nDruP/lesson03/list_lab.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+def fruit_series1():
+"""
+Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+Display the list (plain old print() is fine…).
+Ask the user for another fruit and add it to the end of the list.
+Display the list.
+Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis). Remember that Python uses zero-based indexing, so you will need to correct.
+Add another fruit to the beginning of the list using “+” and display the list.
+Add another fruit to the beginning of the list using insert() and display the list.
+Display all the fruits that begin with “P”, using a for loop.
+"""
```

Comment:
your doc strings should be indented over to align with `fruits` defined below.

I like the idea of putting assignment description in the doc 👍